src/main/webapp/WEB-INF/views/cars/addDriverToCar.jsp

```diff
@@ -0,0 +1,14 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add driver to car</title>
+</head>
+<body>
+<h1>Add driver to car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    Car id: <input type="text" name="car_id"><br>
+    Driver id: <input type="text" name="driver_id"><br>
+    <button type="submit">CREATE</button>
+</form>
```

Comment:
<img width="960" alt="image" src="https://user-images.githubusercontent.com/58032907/204240094-5b337016-6552-4310-9ed0-bfb9a136d297.png">

Reply:
changed it everywhere